api/pkg/cli/hub/get_resource_version.go

```diff
@@ -28,8 +28,13 @@ type resVersionsResponse = rclient.VersionsByIDResponseBody
 // ResVersions is the data in API response consisting of list of versions
 type ResVersions = rclient.VersionsResponseBody
 
+type ResourceVersionResult interface {
+	ResourceVersions() (*ResVersions, error)
+	UnmarshalData() error
+}
+
 // ResourceVersionResult defines API response
-type ResourceVersionResult struct {
+type THResourceVersionResult struct {
```

Comment:
should we rename this as well to `TektonHubResourceVersionResult` ?

Reply:
LGTM, updated